app/controllers/items_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ItemsController < ApplicationController
+  def index
+    if params[:category] == "food"
```

Comment:
You should be able to use your active record associations and queries as to not hard code this. What if you had 20+ categories. You should be using something like `@category.items` to get the items associated with a category.